nodejs/week3/meal/src/backend/api/meals.js

```diff
@@ -0,0 +1,209 @@
+const express = require("express");
+const router = express.Router();
+const knex = require("../database");
+
+const addMeal = async ({ body }) => {
+  const { title, description, maxNumberOfGuests, createdAt, price } = body;
+  return await knex("meals").insert({
+    title: title,
```

Comment:
Good. 
You can also use object desctructuring here, but it is also good without. :) 
Like this:
```
.....insert({
   title, description, numberOfGuests, createdAt, price})
```